source/sass/patterns/molecules/error.scss

```diff
@@ -0,0 +1,42 @@
+@import "../../mixins/media-query";
+@import "../../mixins/sizes";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+@import "../../settings/baselinegrid";
+@import "../../settings/grid";
+
+.error {
+  text-align: center;
+  @include max-inline-size($grid-max-width);
+  @include margin(auto, inline);
+  @include padding($grid-column_gap, inline);
+
+  @include block-spacing($start: $baselinegrid-space-large, $end: $baselinegrid-space-large);
+  @include mq($from: medium) {
+    @include block-spacing($start: $baselinegrid-space-extra_large, $end: $baselinegrid-space-extra_large);
+  }
+
+  @supports (display: flex) {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
```

Comment:
Not for now, but might be more flexible to add a wrapper, zero the block margins and set a `flex-basis` with the desired minimum height the wrapper, with a `flex-shrink` of 0. Would have to have a play.

One for later.